app/enterprise/1.3-x/brain-immunity/install-configure.md

```diff
@@ -3,207 +3,154 @@ title: Kong Brain & Kong Immunity Installation and Configuration Guide
 ---
 
 
-### Introduction
 **Kong Brain** and **Kong Immunity** help automate the entire API and service development life cycle. By automating processes for configuration, traffic analysis and the creation of documentation, Kong Brain and Kong Immunity help organizations improve efficiency, governance, reliability and security. Kong Brain automates API and service documentation and Kong Immunity uses advanced machine learning to analyze traffic patterns to diagnose and improve security. 
 
-### Overview
-
-#### Kong Collector Plugin
-
-Kong Collector Plugin (Collector) configuration includes:
-
-* Kong Collector Plugin is included with Kong Enterprise, and used by Kong Brain and (optional) Kong Immunity.
-* Configure the Collector, and Kong Brain is automatically enabled.
-* If you have Kong Immunity, configure the Collector and Kong Immunity is automatically enabled. 
-
-#### Kong Brain
-
-Kong Brain (Brain) features include:
-
-* Kong Brain uses the real-time Kong Collector Plugin.
-* Once the Collector is configured, Kong Brain is automatically enabled. 
-* Kong Brain ingests your documentation and data flows, analyze changes and take action.
-* Users can create workflows and approve changes directly in Kong Manager.
-* Visually map services using the real-time Kong Service Map. 
-
-#### Kong Immunity
-
-Kong Immunity (Immunity) features include:
-
-* Kong Immunity is optional.
-* Once the Collector is configured, Kong Immunity is automatically enabled. 
-* View Kong Immunity alerts in the Kong Service Map and take action with just a few clicks.
-* Instantly diagnose issues. Receive Slack alerts notifying users of anomalies that require attention. 
-
-#### Kong Service Map
-
-* For information about the Kong Service Map, see **Using Kong’s Service Map**.
+When you purchase Kong Brain and/or Kong Immunity, you obtain both add-ons together or individually:
+* Kong Brain and Kong Immunity
+* Kong Brain
+* Kong Immunity
 
+![Kong Brain and Kong Immunity] (https://doc-assets.konghq.com/1.3/brain_immunity/KongBrainImmunity_overview.png)
```

Comment:
```suggestion
![Kong Brain and Kong Immunity](https://doc-assets.konghq.com/1.3/brain_immunity/KongBrainImmunity_overview.png)
```

Reply:
The image goes further down in the docs.